common/autoware_ad_api_msgs/README.md

```diff
@@ -1,17 +1,25 @@
 # autoware_ad_api_msgs
 
-## InterfaceVersion
+## ResponseStatus
+
+This message is a response status commonly used in the service type API. For details, see [the response status][docs-response-status].
+Each API defines its own error code and assigns a level to them. For example, define 12345 as the code to indicate that the parameter is invalid.
+The level of this code is an error if the process cannot be continued, or a warning if the process can be continued by using the default value or something.
```

Comment:
> a warning if the process can be continued by using the default value or something.

Could you elaborate on this part? I'd like to know a bit more concrete examples.
I couldn't understand what is `the default value or something` and when we should use them.